model/Metadata/Service/AdvancedSearchSettingsService.php

```diff
@@ -32,11 +32,15 @@
 use oat\tao\model\search\Contract\SearchSettingsServiceInterface;
 use oat\tao\model\search\ResultColumn;
 use oat\tao\model\search\SearchSettings;
+use oat\tao\model\TaoOntology;
 
 class AdvancedSearchSettingsService implements SearchSettingsServiceInterface
 {
     private const OMIT_PROPERTIES = [
-        OntologyRdfs::RDFS_LABEL
+        OntologyRdfs::RDFS_LABEL,
```

Comment:
@shpran we probably will need to play with this based on FF, so we will hide uniqueId and other translations properties if the feature flags is not ON:

- FEATURE_FLAG_TRANSLATION_ENABLED
- FEATURE_FLAG_UNIQUE_NUMERIC_QTI_IDENTIFIER